projects/packages/identity-crisis/src/class-ui.php

```diff
@@ -95,24 +95,26 @@ private static function get_initial_state() {
 	 * @return array
 	 */
 	private static function get_initial_state_data() {
-		$idc_urls       = Identity_Crisis::get_mismatched_urls();
-		$current_screen = get_current_screen();
-		$is_admin       = current_user_can( 'jetpack_disconnect' );
+		$idc_urls                  = Identity_Crisis::get_mismatched_urls();
+		$current_screen            = get_current_screen();
+		$is_admin                  = current_user_can( 'jetpack_disconnect' );
+		$dynamic_site_url_detected = Identity_Crisis::detect_dynamic_site_url();
```

Comment:
This line is the new value being added & passed down, the rest is linter spacing.